packages/docker-engine/.gitignore

```diff
@@ -0,0 +1 @@
+/*.patch.bz2
```

Comment:
This ignore isn't specific to docker-engine, it would apply to any package, so I think this file should be at `/packages/.gitignore` instead.

Reply:
Yeah, we can move it up to the root of packages. :)